### ISSUE TITLE
unit passed to .time() function ignored.

### ISSUE THREAD

**Issue by joscha** [Krb686/nanotimer#20]
When using `.time` the runtime passed to the callback function is not based on the unit given.

```javascript
var timer = require("nanotimer");

new timer().time(function(end) {
    setTimeout(end, 2000);
}, '', 'm', function(runtime) { console.log('Runtime was: ' + runtime + 'ms.'); });
```

This code uses `m` (according to the docs the unit for ms). However the callback is something along:
```
Runtime was: 1.62894ms.
```

It seems as if the unit is not picked up correctly.
You can try the above code here: https://tonicdev.com/npm/nanotimer

**Comment by Krb686** [comment]
Yikes, not sure how that got missed.  Thanks, fixed by https://github.com/Krb686/nanotimer/commit/886f9051dbd3571ce442b0896d4b6fa240dc80b9

**Comment by joscha** [comment]
Awesome, thank you! Any chance you'd release a new version with the fix?

**Comment by Krb686** [comment]
Yep, of course. Just published 0.3.13

**Comment by joscha** [comment]
Thank you!
